docs/tutorials/monitoring/index.md

```diff
@@ -59,19 +59,20 @@ step.
     ```bash
     ./jaeger-all-in-one
     ```
-3. **Enable tracing in a Feldera pipeline**
-    - You can also enable/disable tracing and specify the Jaeger endpoint in the pipeline configuration:
+3. **Tracing a Feldera pipeline**
+    - Enable/disable tracing and specify the Jaeger endpoint in the pipeline configuration:
    ```bash
    curl -i -X PUT http://localhost:8080/v0/pipelines/tracing-pipeline \
       -H 'Content-Type: application/json' \
       -d '{
         "description": "Example pipeline",
         "program_name": "example-program",
-        "config": {"tracing": true, "tracing_endpoint_jaeger": "tracing.feldera.io:6831"},
+        "config": {"tracing": true, "tracing_endpoint_jaeger": "host.docker.internal:6831"},
```

Comment:
docker comes in with a builtin jaeger tracing endpoint?

Reply:
With this you can run the tracing software on your machine and the pipeline can still reach it from within the container.